py-polars/tests/unit/operations/test_groupby.py

```diff
@@ -538,7 +538,7 @@ def test_groupby_dynamic_iter() -> None:
     result2 = [
         (name, data.shape)
         for name, data in df.groupby_dynamic(
-            "datetime", every="1h", closed="left", by="a"
+            "datetime", every=timedelta(hours=1), closed="left", by="a"
```

Comment:
Does it still work with the `"1h"` syntax?

Reply:
Of course ;)  I was just extending a little bonus coverage there - a few lines above it uses "1h" in the same param.

Reply:
Thanks, it wasn't clear to me from the code. Thanks a lot. :)

P.S. drinking your coffee atm, while completing out of core sort. :sunglasses: :

Reply:
Haha, nice! I hope your coffee-loving friend approved of it too... ;)